www/index.html

```diff
@@ -4,9 +4,12 @@
     <meta charset="utf-8">
     <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
     <meta http-equiv="Content-Security-Policy"
-        content="default-src 'self' data: gap: https://ssl.gstatic.com http://nominatim.openstreetmap.org https://e-mission.eecs.berkeley.edu 'unsafe-inline';
+        content="default-src 'self' data: gap: https://ssl.gstatic.com http://nominatim.openstreetmap.org https://e-mission.eecs.berkeley.edu https://habitica-assets.s3.amazonaws.com https://www.habitica.com 
+        https://habitica.com/api/v3/members/4f369eef-aed4-4408-bcbf-b34896daf7e3 'unsafe-inline';
```

Comment:
Do you need this? Isn't it subsumed by the `https://www.habitica.com`?

The reason I ask is that it looks like this is the API to read data/assets for member `4f369eef-aed4-4408-bcbf-b34896daf7e3`. Since each user will have their own habitica UUID, this is not a scalable solution.

So it would be nice to know whether it is necessary or not.